generators/client/templates/angular/src/test/javascript/spec/app/account/settings/_settings.component.spec.ts

```diff
@@ -0,0 +1,125 @@
+import {ComponentFixture, TestBed, async, inject} from '@angular/core/testing';
+import {MockBackend} from '@angular/http/testing';
+import {Http, BaseRequestOptions} from '@angular/http';
+import {Observable} from 'rxjs/Rx';
+import {<%=jhiPrefixCapitalized%>LanguageService} from 'ng-jhipster';
+import {Principal, AccountService, <%=jhiPrefixCapitalized%>LanguageHelper} from '../../../../../../main/webapp/app/shared';
+import {SettingsComponent} from "../../../../../../main/webapp/app/account/settings/settings.component";
+import {MockLanguageService} from "../../../helpers/language.service";
+import {MockAccountService} from "../../../helpers/account.service";
+import {MockPrincipal} from "../../../helpers/principal.service";
+<%_ if (websocket === 'spring-websocket') { _%>
+import {<%=jhiPrefixCapitalized%>TrackerService} from "../../../../../../main/webapp/app/shared/tracker/tracker.service";
+import {MockTrackerService} from "../../../helpers/tracker.service";
+<%_ } _%>
+
+
+describe('Component Tests', () => {
+
+    describe('SettingsComponent', () => {
+
+        let comp:SettingsComponent;
+        let fixture:ComponentFixture<SettingsComponent>;
+        let mockAuth:MockAccountService;
+        let mockPrincipal:MockPrincipal;
+
+        beforeEach(async(() => {
+            TestBed.configureTestingModule({
+                declarations: [SettingsComponent],
+                providers: [MockBackend,
+                    {
+                        provide: Principal,
+                        useClass: MockPrincipal
+                    },
+                    {
+                        provide: AccountService,
+                        useClass: MockAccountService
+                    },
+                    <%_ if (websocket === 'spring-websocket') { _%>
+                    {
+                        provide: <%=jhiPrefixCapitalized%>TrackerService,
+                        useClass: MockTrackerService
+                    },
+                    <%_ } _%>
+                    BaseRequestOptions,
+                    {
+                        provide: Http,
+                        useFactory: (backendInstance:MockBackend, defaultOptions:BaseRequestOptions) => {
+                            return new Http(backendInstance, defaultOptions);
+                        },
+                        deps: [MockBackend, BaseRequestOptions]
+                    },
+                    {
+                        provide: <%=jhiPrefixCapitalized%>LanguageService,
+                        useClass: MockLanguageService
+                    },
+                    {
+                        provide: <%=jhiPrefixCapitalized%>LanguageHelper,
```

Comment:
same comment:
I think `<%=jhiPrefixCapitalized%>LanguageHelper` is only need when enableTranslation